### PR TITLE
Fix/missing alt for profile image

### PR DESCRIPTION
**Pull request by mikeslinkman**
## Description

Adds the alt attribute for the avatar of a user based on the users name.

## Visual changes

No visual changes

## Functional changes

- [x] Code style has been fixed by running the `composer cs` command.
- [x] Changes have been tested to not break existing functionality.
- [x] Documentation is up-to-date.

### CODE REVIEW

**Comment by zepfietje** [comment]
Wouldn't it be ok to just use the user name without any prefix?
Also, this should probably be added to the tenant avatar in the tenant menu too.

**Comment by mikeslinkman** [comment]
> Wouldn't it be ok to just use the user name without any prefix? Also, this should probably be added to the tenant avatar in the tenant menu too.

It would be an improvement compared to the current situation, but we would leave the user "guessing" what the context for the image is. Ideally we would add the additional context, and I would be willing to add more translations by using google translate but first wanted to get a initial response. I'll add it for the tenant variant as well once we've settled on the way we will handle the translations (if we handle these)

** Edit **

Just discussed internally with our Accessibility Engineer and he told me the best would be to do it like this:

```
Avatar of [Name] // EN
Avatar van [Name] // NL
```

When we add alt text in these situations for our own projects (also for logo purposes of the company) we normally do it like this `[Company Name] logo` compared to `[Comapny Name]` just adds a little more context for people.

**Comment by zepfietje** [comment]
Thanks for clarifying, @mikeslinkman.
I agree, though we should discuss if avatar / profile picture / profile photo or something else fits best in this case. Kinda tricky since it may depend on the actual application.

Also, after this PR, when we add it to tenant avatars too, we should also update `packages/panels/resources/views/components/logo.blade.php` to use a "logo" suffix.

**Comment by mikeslinkman** [comment]
> Thanks for clarifying, @mikeslinkman. I agree, though we should discuss if avatar / profile picture / profile photo or something else fits best in this case. Kinda tricky since it may depend on the actual application.
> 
> Also, after this PR, when we add it to tenant avatars too, we should also update `packages/panels/resources/views/components/logo.blade.php` to use a "logo" suffix.

Yes, I can totally add it over there as well. I'm open to any of those but I have a slight preference for using 'Avatar' since this is also the best description when it is not an actual photo but instead shows the initials of the authenticated user. But also happy to explore other options

**Comment by danharrin** [comment]
+1 for avatar, its the terminology we use internally anyway

File: packages/panels/resources/views/components/avatar/user.blade.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 <x-filament::avatar
     :src="filament()->getUserAvatarUrl($user)"
+    :alt="__('filament-panels::layout.top_bar.avatar', ['user' => filament()->getUserName($user)])"
```
Comment (danharrin) [inline packages/panels/resources/views/components/avatar/user.blade.php]: Can you refactor the translation to use `filament-panels::layout.avatar.alt` please?

File: packages/panels/resources/lang/nl/layout.php [hunk 2]
```diff
@@ -52,4 +52,10 @@
 
     ],
 
+    'top_bar' => [
+
+        'avatar' => 'Avatar vertegenwoordigt :user',
+
+    ],
```
Comment (danharrin) [inline packages/panels/resources/lang/nl/layout.php]: Please remove new lines in arrays when there is no nesting, as per the other instances in these files

**Review by danharrin — CHANGES_REQUESTED** [review]
While this is open, please also apply this so the tenant avatar, which can use the same translation string

**Comment by zepfietje** [comment]
> While this is open, please also apply this so the tenant avatar, which can use the same translation string

And add the "logo" suffix/prefix to the brand logo component alt.

**Comment by danharrin** [comment]
For that one, the translation string can be layout.logo.alt I think

File: packages/panels/resources/lang/en/layout.php [hunk 3]
```diff
@@ -52,4 +52,12 @@
 
     ],
 
+    'avatar' => [
+        'alt' => 'Avatar representing :user',
```
Comment (zepfietje) [inline packages/panels/resources/lang/en/layout.php]:
```suggestion
        'alt' => 'Avatar of :user',
```

File: packages/panels/resources/lang/nl/layout.php [hunk 4]
```diff
@@ -50,6 +50,12 @@
 
         ],
 
-    ],
+        'avatar' => [
+            'alt' => 'Avatar vertegenwoordigt :user',
```
Comment (zepfietje) [inline packages/panels/resources/lang/nl/layout.php]:
```suggestion
            'alt' => 'Avatar van :user',
```

File: packages/panels/resources/lang/en/layout.php [hunk 5]
```diff
@@ -52,4 +52,12 @@
 
     ],
 
+    'avatar' => [
+        'alt' => 'Avatar of :user',
```
Comment (zepfietje) [inline packages/panels/resources/lang/en/layout.php]:
```suggestion
        'alt' => 'Avatar of :name',
```

File: packages/panels/resources/lang/nl/layout.php [hunk 6]
```diff
@@ -50,6 +50,12 @@
 
         ],
 
-    ],
+        'avatar' => [
+            'alt' => 'Avatar van :user',
```
Comment (zepfietje) [inline packages/panels/resources/lang/nl/layout.php]:
```suggestion
            'alt' => 'Avatar van :name',
```

File: packages/panels/resources/views/components/avatar/tenant.blade.php [hunk 7]
```diff
@@ -4,6 +4,7 @@
 
 <x-filament::avatar
     :circular="false"
+    :alt="__('filament-panels::layout.avatar.alt', ['user' => filament()->getTenantName($tenant)])"
```
Comment (zepfietje) [inline packages/panels/resources/views/components/avatar/tenant.blade.php]:
```suggestion
    :alt="__('filament-panels::layout.avatar.alt', ['name' => filament()->getTenantName($tenant)])"
```

File: packages/panels/resources/views/components/avatar/tenant.blade.php [hunk 8]
```diff
@@ -4,6 +4,7 @@
 
 <x-filament::avatar
     :circular="false"
+    :alt="__('filament-panels::layout.avatar.alt', ['name' => filament()->getTenantName($tenant)])"
     :src="filament()->getTenantAvatarUrl($tenant)"
```
Comment (zepfietje) [inline packages/panels/resources/views/components/avatar/tenant.blade.php]:
```suggestion
    :src="filament()->getTenantAvatarUrl($tenant)"
    :alt="__('filament-panels::layout.avatar.alt', ['name' => filament()->getTenantName($tenant)])"
```

File: packages/panels/resources/views/components/avatar/user.blade.php [hunk 9]
```diff
@@ -4,6 +4,7 @@
 
 <x-filament::avatar
     :src="filament()->getUserAvatarUrl($user)"
+    :alt="__('filament-panels::layout.avatar.alt', ['user' => filament()->getUserName($user)])"
```
Comment (zepfietje) [inline packages/panels/resources/views/components/avatar/user.blade.php]:
```suggestion
    :alt="__('filament-panels::layout.avatar.alt', ['name' => filament()->getUserName($user)])"
```

File: packages/panels/resources/lang/en/layout.php [hunk 10]
```diff
@@ -52,4 +52,12 @@
 
     ],
 
+    'avatar' => [
+        'alt' => 'Avatar of :name',
+    ],
+
+    'logo' => [
+        'alt' => ':brand logo',
```
Comment (danharrin) [inline packages/panels/resources/lang/en/layout.php]:
```suggestion
        'alt' => ':name logo',
```

File: packages/panels/resources/lang/nl/layout.php [hunk 11]
```diff
@@ -50,6 +50,12 @@
 
         ],
 
-    ],
+        'avatar' => [
+            'alt' => 'Avatar van :name',
+        ],
 
+        'logo' => [
+            'alt' => ':brand logo',
```
Comment (danharrin) [inline packages/panels/resources/lang/nl/layout.php]:
```suggestion
            'alt' => ':name logo',
```

File: packages/panels/resources/views/components/logo.blade.php [hunk 12]
```diff
@@ -28,7 +28,7 @@
         </div>
     @elseif (filled($logo))
         <img
-            alt="{{ $brandName }}"
+            alt="{{ __('filament-panels::layout.logo.alt', ['brand' => $brandName]) }}"
```
Comment (danharrin) [inline packages/panels/resources/views/components/logo.blade.php]:
```suggestion
            alt="{{ __('filament-panels::layout.logo.alt', ['name' => $brandName]) }}"
```

**Comment by danharrin** [comment]
Thanks @mikeslinkman

**Pull request merged**